data/implementations/utils.json

```
@@ -1,17 +1,13 @@
 {
   "id": "utils",
-  "title": "General Purpose Utils & Datatypes",
+  "title": "General Purpose Utilities & Datatypes",
+  "interface": "https://github.com/libp2p/js-libp2p-interfaces",
   "libs": [
     {
       "id": "crypto",
```

```suggestion
      "id": "libp2p-crypto",
```
Any reason this doesn't follow the naming convention of others?